tests/test_philips.py

```diff
@@ -0,0 +1,45 @@
+"""Tests for Philips RDM001 and RDM004 quirks."""
+import pytest
+
+from tests.common import ClusterListener
+
+import zhaquirks
+from zhaquirks.philips.rdm001 import PhilipsRDM001
+from zhaquirks.philips.rdm004 import PhilipsRDM004
+
+zhaquirks.setup()
+
+#@pytest.mark.parametrize("manufacturer,model",
+#    [
+#        ("Philips", "RDM001"),
+#        ("Signify Netherlands B.V.", "RDM001"),
+#    ],
+#)
+
+def test_RDM001_signature(
+    manufacturer: str, model: str, assert_signature_matches_quirk
+) -> None:
+    """Test that the signature of all supported RDM devices is matched to its quirk."""
+    signature = {
+        "node_descriptor": "<SimpleDescriptor endpoint=1 profile=260 device_type=2080 device_version=1 input_clusters=[0, 1, 3, 64512] output_clusters=[3, 4, 6, 8, 25]>",
+        "endpoints": {
+            "1": {
+                "profile_id": 260,
+                "device_type": "0x820",
+                "in_clusters": [
+                    "0x0",
+                    "0x1",
+                    "0x3",
+                    "0xFC00",
+                ],
+                "out_clusters": ["0x3", "0x4", "0x6", "0x8", "0x19"],
+            }
+        },
+        "manufacturer": "Philips",
+        "model": "RDM001",
+        "class": "zhaquirks.philips.rdm001.PhilipsRDM001",
+    }
+
+    assert_signature_matches_quirk(
+        zhaquirks.philips.rdm001.PhilipsRDM001, signature
+    ) 
```

Comment:
Its one space after the ) that making the test complaining.

Reply:
The other:
```
+++ b/tests/test_philips.py
@@ -1,11 +1,7 @@
 """Tests for Philips RDM001 and RDM004 quirks."""
-import pytest
 
-from tests.common import ClusterListener
 
 import zhaquirks
-from zhaquirks.philips.rdm001 import PhilipsRDM001
-from zhaquirks.philips.rdm004 import PhilipsRDM004
 
 zhaquirks.setup()
```
Its  looks the test i not liking importing this function but i dont  knowing if its needed for the code or not then im not one code warier.
I think Puddly or TheJullian is knowing it bettter.

Reply:
nope, the space after the ) was not the problem apparently

Reply:
The first error is now gone only the import section and the the strange last ones . . . so one step forewords and . . .